waku/v2/node/wakunode2.nim

```diff
@@ -155,7 +144,7 @@ proc init*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   #    msgIdProvider = msgIdProvider,
   #    triggerSelf = true, sign = false,
   #    verifySignature = false).PubSub
-  result = WakuNode(
+  return WakuNode(
```

Comment:
Nitpick but I'd probably first init it then `return wakuNode`. Just because we are creating a new peer manager etc, less stuff going on in one expression/statement

Reply:
Done. 47981cf